src/google/protobuf/compiler/php/php_generator.cc

```diff
@@ -1625,38 +1625,37 @@ static std::string EscapePhpdoc(const std::string& input) {
   std::string result;
   result.reserve(input.size() * 2);
 
-  char prev = '*';
+  char prev = '\0';
 
   for (std::string::size_type i = 0; i < input.size(); i++) {
     char c = input[i];
     switch (c) {
-      case '*':
-        // Avoid "/*".
-        if (prev == '/') {
-          result.append("&#42;");
-        } else {
-          result.push_back(c);
-        }
-        break;
+      // "/*" is allowed, do nothing
```

Comment:
Please remove this commented-out code. You could leave a comment that "/*" is allowed so we do not translate it.

Reply:
done!